docs/rules/index.md

```diff
@@ -4,108 +4,106 @@ sidebarDepth: 0
+| [prefer-named-capture-group](https://ota-meshi.github.io/eslint-plugin-regexp/rules/prefer-named-capture-group.html)             | enforce using named capture groups                                     |    |    |    |    |    |
+| [prefer-named-replacement](https://ota-meshi.github.io/eslint-plugin-regexp/rules/prefer-named-replacement.html)                 | enforce using named replacement                                        |    |    | 🔧 |    |    |
+| [prefer-plus-quantifier](https://ota-meshi.github.io/eslint-plugin-regexp/rules/prefer-plus-quantifier.html)                     | enforce using `+` quantifier                                           | ✅  |    | 🔧 |    |    |
+| [prefer-question-quantifier](https://ota-meshi.github.io/eslint-plugin-regexp/rules/prefer-question-quantifier.html)             | enforce using `?` quantifier                                           | ✅  |    | 🔧 |    |    |
+| [prefer-result-array-groups](https://ota-meshi.github.io/eslint-plugin-regexp/rules/prefer-result-array-groups.html)             | enforce using result array `groups`                                    |    |    | 🔧 |    |    |
+| [prefer-star-quantifier](https://ota-meshi.github.io/eslint-plugin-regexp/rules/prefer-star-quantifier.html)                     | enforce using `*` quantifier                                           | ✅  |    | 🔧 |    |    |
+| [prefer-t](https://ota-meshi.github.io/eslint-plugin-regexp/rules/prefer-t.html)                                                 | enforce using `\t`                                                     |    |    | 🔧 |    | ❌  |
+| [prefer-unicode-codepoint-escapes](https://ota-meshi.github.io/eslint-plugin-regexp/rules/prefer-unicode-codepoint-escapes.html) | enforce use of unicode codepoint escapes                               | ✅  |    | 🔧 |    |    |
+| [prefer-w](https://ota-meshi.github.io/eslint-plugin-regexp/rules/prefer-w.html)                                                 | enforce using `\w`                                                     | ✅  |    | 🔧 |    |    |
+| [sort-character-class-elements](https://ota-meshi.github.io/eslint-plugin-regexp/rules/sort-character-class-elements.html)       | enforces elements order in character class                             |    |    | 🔧 |    |    |
+| [sort-flags](https://ota-meshi.github.io/eslint-plugin-regexp/rules/sort-flags.html)                                             | require regex flags to be sorted                                       | ✅  |    | 🔧 |    |    |
+| [unicode-escape](https://ota-meshi.github.io/eslint-plugin-regexp/rules/unicode-escape.html)                                     | enforce consistent usage of unicode escape or unicode codepoint escape |    |    | 🔧 |    |    |
 
-### Deprecated
```

Comment:
I want deprecated to be a separate list just like the list in ESLint core. Can we do that?
https://eslint.org/docs/latest/rules/#deprecated

Reply:
I just filed https://github.com/bmish/eslint-doc-generator/issues/291 which will be necessary to achieve that. We can only use `--split-by` to split the rules list by one property currently, and we're already using that for `--split-by meta.docs.category`. Note that eslint-plugin-jest has an example with a [rules list](https://github.com/jest-community/eslint-plugin-jest#rules) where we use `--split-by meta.deprecated`.

Reply:
I have solved this by defining a `ruleListSplit` function in the config file using the new feature of the [`--rule-list-split` config option](https://github.com/bmish/eslint-doc-generator#configuration-options) from eslint-doc-generator [1.2.0](https://github.com/bmish/eslint-doc-generator/releases/tag/v1.2.0). The new sections should now match the old sections.

Reply:
Looks good. Thank you!